src/App.tsx

```diff
@@ -1,5 +1,6 @@
 import logo from "./logo.svg";
 import "./App.css";
+import React, {useState} from "react";
```

Comment:
This is no longer necessary, since we have auto-imports installed
more info: https://github.com/antfu/unplugin-auto-import

Reply:
![Screen Shot 2021-12-10 at 1 04 29 PM](https://user-images.githubusercontent.com/43630417/145620799-f32bc2bd-1e79-49dc-a8fd-95924cebe2d6.png)

if I remove that line the app still works but I get this ugly red line

Reply:
Currently, [unplugin-auto-import](https://github.com/antfu/unplugin-auto-import) has support for all these hooks: [Hooks presets](https://github.com/antfu/unplugin-auto-import/blob/main/src/presets/react.ts)

Reply:
Maybe it is a problem with VSCode, it does not affect our development, we will look for a visual solution for that